Set cards=0 to the default configuration so that autoload can load it without
stopping the startup process of Asterisk.
Of course, a fix for the code was needed to bail-out on cards=0 :-)

 -- Faidon Liambotis <paravoid@debian.org>

--- asterisk-1.4.15~dfsg.orig/channels/chan_vpb.cc
+++ asterisk-1.4.15~dfsg/channels/chan_vpb.cc
@@ -2883,7 +2883,13 @@ static enum ast_module_load_result load_
 		v = ast_variable_browse(cfg, "general");
 		while (v){
 			if (strcasecmp(v->name, "cards") == 0) {
-				ast_log(LOG_NOTICE,"VPB Driver configured to use [%d] cards\n",atoi(v->value));
+				if (atoi(v->value) == 0) {
+					ast_log(LOG_ERROR,"VPB Driver not configured\n");
+					error = AST_MODULE_LOAD_DECLINE;
+					goto done;
+				} else {
+					ast_log(LOG_NOTICE,"VPB Driver configured to use [%d] cards\n",atoi(v->value));
+				}
 			}
 			else if (strcasecmp(v->name, "indication") == 0) {
 				use_ast_ind = 1;
--- asterisk-1.4.15~dfsg.orig/configs/vpb.conf.sample
+++ asterisk-1.4.15~dfsg/configs/vpb.conf.sample
@@ -8,7 +8,7 @@
 ;
 ; Total number of Voicetronix cards in this machine
 ;
-cards=1
+cards=0
 
 ;
 ; Which indication functions to use
